nextjs-sdk/package.json

```diff
@@ -1,8 +1,9 @@
 {
-  "name": "@pcc/react",
+  "name": "@pcc/react-sdk",
```

Comment:
Updating this since our NPM package is called as `pcc-react-sdk`.
With this update from this PR, CLI command would be called as
```
react-sdk init <project-directory>
```
Let me know if this looks good.
@kevinstubbs @a11rew